MapboxAndroidDemo/src/global/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -297,22 +296,19 @@ public void onItemClicked(RecyclerView recyclerView, int position, View view) {
 
       analytics.clickedOnIndividualExample(getString(model.getTitle()), loggedIn);
       analytics.viewedScreen(getString(model.getTitle()), loggedIn);
-
     }
   }
 
   @Override
   protected void onResume() {
     super.onResume();
-    Log.d(TAG, "onResume: ");
-    ItemClickSupport.addTo(recyclerView).setOnItemClickListener(this);
+    itemClickSupport.setOnItemClickListener(this);
   }
 
   @Override
   protected void onPause() {
     super.onPause();
-    Log.d(TAG, "onPause: ");
-    ItemClickSupport.removeFrom(recyclerView);
+    itemClickSupport.setOnItemClickListener(null);
```

Comment:
Is this enough to resolve the issue? I was testing with this invocation in `onItemClicked`.

Reply:
```java
@Override
  protected void onPause() {
    super.onPause();
    itemClickSupport.setOnItemClickListener(null);
  }
```

is working 💯 for me